docs/guide/cleanup.md

```diff
@@ -131,3 +131,23 @@ The fix here is often that we either:
 - Move to a dependency which is leaner and is more focused.
 
 In this situation, it is worth looking for a lighter (and hopefully faster) alternative.
+
+### Finding Other Dependents of Potential Packages
```

Comment:
i think this should be:

> Tip: Finding dependents of a package

Since the previous 3 sections are directly connected (i.e. discover -> improve -> fix), but this one won't be.

if we name it `Tip:`, it opens the door for us adding other tips in future

Reply:
Replaced it. Its all orange text now :D.
I dont mind it. Wdyt?

Reply:
i'd be tempted to keep it as a section since we might add more of them, and these tip boxes aren't visible in the contents list (the sidebar)

Reply:
Oh you meant to name the whole section "Tip: blabla"?

Reply:
done